packages/testing/playwright/tests/cli-workflows/workflows/260.json

```
@@ -0,0 +1,158 @@
+{
+	"createdAt": "2021-01-21T10:32:07.628Z",
+	"updatedAt": "2021-06-04T10:25:41.024Z",
+	"id": "260",
+	"name": "Tools within loops",
+	"active": false,
+	"nodes": [
+		{
+			"parameters": {},
+			"type": "n8n-nodes-base.manualTrigger",
+			"typeVersion": 1,
+			"position": [-800, 992],
+			"id": "fa8ff0c4-b035-4c92-bdc2-033f903fca69",
+			"name": "When clicking ‘Test workflow’"
+		},
+		{
+			"parameters": {
+				"jsCode": "const result = [\n  'n8n',\n  'make',\n  'zappier',\n  'workato'\n].map(i => ({ json: {name: i}}))\n\nreturn result;"
+			},
+			"type": "n8n-nodes-base.code",
+			"typeVersion": 2,
+			"position": [-592, 992],
+			"id": "b82a0103-35c0-4f5c-9cd5-8c6b4e5e5f8a",
+			"name": "Code"
+		},
+		{
+			"parameters": {
+				"promptType": "define",
+				"text": "=Search HackerNews for {{ $json.name }}",
+				"options": {}
+			},
+			"type": "@n8n/n8n-nodes-langchain.agent",
+			"typeVersion": 1.9,
```

Should we test the latest version of agent node?

```suggestion
			"typeVersion": 2.2,
```